tests/config/debugControllerBackend.ts

```diff
@@ -135,46 +136,21 @@ export class Backend extends EventEmitter {
     };
   }
 
-  async initialize() {
-    await this._send('initialize', { codegenId: 'playwright-test', sdkLanguage: 'javascript' });
+  channel(): channels.DebugControllerChannel {
+    return new Proxy(this, {
+      get: (target, propKey) => {
+        const origMethod = target[propKey];
+        if (typeof origMethod === 'function')
```

Comment:
nit: I feel like explicit list, which is just `on` in this case, would be easier to understand.